internal/pkg/skuba/deployments/ssh/ssh.go

```diff
@@ -237,6 +242,14 @@ func (t *Target) initClient() error {
 			klog.Errorf("ssh authentication error: please make sure you have added to "+
 				"your ssh-agent a ssh key that is authorized in %q.", t.target.Target)
 			return errSSHAuthErr
+		} else if strings.Contains(err.Error(), "connection refused") {
+			klog.Errorf("ssh error: please make sure you have the correct IP/FQDN to "+
```

Comment:
connection refused & no such host with the same klog output, how to differentiate these?

Reply:
"connection refused" showed when ip address was not reachable.
" no such host" showed when ip was like "10.86.0", which was not complete IP.

Reply:
But the klog errorf both are “ssh error: please make sure you have the correct IP/FQDN to “. How user can distinguish them?

Reply:
We do not distinguish the message because this all caused incorrect ip/Fqdn. So the user need to check ip or fqdn at the end of message.
Error message will be like
```
 ssh error: please make sure you have the correct IP/FQDN to 10.86.0.
```

Then, user should notice that he/she typo up address

Reply:
`connection refused` might be caused by the firewall?

Reply:
Yes, that is true.